examples/ps.go

```diff
@@ -1,17 +1,17 @@
-package examples
+package main
 
 import (
 	"fmt"
 
-	"github.com/cloudfoundry/gosigar"
+	sigar "github.com/cloudfoundry/gosigar"
 )
 
-func ps() {
+func main() {
 	pids := sigar.ProcList{}
 	pids.Get()
 
 	// ps -eo pid,ppid,stime,time,rss,state,comm
-	fmt.Print("  PID  PPID STIME     TIME    RSS S COMMAND\n")
+	fmt.Print("   PID   PPID STIME     TIME    RSS S COMMAND\n")
```

Comment:
What is this change intended to do? When I run the unmodified code, the `D` in `PID` lines up with the right-most digit in each of the PIDs. (Does your system have PIDs that have more than six digits in them? Mine does not, and that might be why I'm not seeing what you're trying to correct.)

If it's trying to exactly mimic the format of the `ps` header in the output of the command above, it doesn't seem to match what I'm seeing on a Ubuntu Jammy Linux system: (Notice how there are four spaces between the start of the line and the `P` character in the header.)
```
$ ps -eo pid,ppid,stime,time,rss,state,comm | head -n1
    PID    PPID STIME     TIME   RSS S COMMAND
$ ps --version
ps from procps-ng 3.3.17
$ uname -a
Linux daily-cheese-ops-manager 5.15.0-56-generic #62-Ubuntu SMP Tue Nov 22 19:54:14 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.1 LTS
Release:	22.04
Codename:	jammy
```
```

Reply:
I get this with this PR:
```
$ go run ps.go 2> /dev/null| head -n1; ps -eo pid,ppid,stime,time,rss,state,comm | head -n1
   PID   PPID STIME     TIME    RSS S COMMAND
    PID    PPID STIME     TIME   RSS S COMMAND
```

So `ps` has even wider columns for the pid and ppid.

This seems to match with your output.

My highest pid is 8 characters long:
```
$ ps -eo pid | tail -1 | wc -c
8
```

I've pushed 1cac03073149bd3d880174248e07a15ef1da39fb to match `ps` more closely.